CHANGELOG.md

```diff
@@ -3,8 +3,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+
+
 ## [Unreleased]
```

Comment:
i'm going to push a commit of my own to remove the extra whitespace in your branch.

```suggestion
This project adheres to [Semantic Versioning](http://semver.org/).

## [Unreleased]
```